test/azul_test_case.py

```diff
@@ -163,7 +164,11 @@ def setUp(self) -> None:
         moto_api_backend.reset()
 
     catalog: CatalogName = 'test'
-    catalog_config = f'hca:{catalog}:metadata/hca:repository/dss'
+    catalog_spec = json.dumps([
```

Comment:
Let's not use "spec" here. Why the rename, anyways?

Reply:
When I had a `from_json` method the input to the method was called `spec`. It made sense to change the name to catalog_spec rather than catalog_config. 

Can revert this.